.github/CODEOWNERS

```diff
@@ -122,6 +122,7 @@
 /rum_react/                              @DataDog/rum-app
 /rum_react_native/                       @DataDog/rum-app
 /rundeck/                                forrest@rundeck.com
+/scalr/                                  @soltysss
```

Comment:
You could also add `scalr` to [this section](https://github.com/DataDog/integrations-extras/blob/master/.github/CODEOWNERS#L169) so our doc team ca be pinged when needed 🙂